bindings/pulp/bindings/server.py

```diff
@@ -223,7 +225,7 @@ def _build_url(self, path, queries=()):
                     of self.path_prefix
         """
         # build the request url from the path and queries dict or tuple
-        if not path.startswith(self.path_prefix):
+        if not path.startswith(self.path_prefix) and not ignore_prefix:
```

Comment:
You've added this ignore_prefix throughout this file without documenting it in any docblocks. Every method that this appears in needs to have it documented.